pkg/workloads/console/controller.go

```diff
@@ -323,6 +323,25 @@ func (r *reconciler) updateStatus(job *batchv1.Job) error {
 		return nil
 	}
 
+	// Console phase from Pending to Running
+	if newStatus.Phase == workloadsv1alpha1.ConsoleRunning &&
+		r.console.Status.Phase == workloadsv1alpha1.ConsolePending {
+		auditLog(r.logger, r.console, nil, "ConsoleStart", "Console started")
```

Comment:
Let's extract these event names to constants, so that we have a nice list at the top

Reply:
Updated